devicetypes/smartthings/Orvibo-Contact-Sensor.src/Orvibo-Contact-Sensor.groovy

```diff
@@ -131,7 +131,7 @@ def configure() {
 	log.debug "Configuring Reporting, IAS CIE, and Bindings."
 	//The electricity attribute is reported without bind and reporting CFG. The TI plan reports the power once in about 10 minutes; the NXP plan reports the electricity once in 20 minutes
 	if (getDataValue("manufacturer") == "Aurora") {
-		cmds = zigbee.iasZoneConfig(30, 60 * 5) + zigbee.batteryConfig()
+		cmds = zigbee.configureReporting(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS, DataType.BITMAP16, 30, 60 * 5, null) + zigbee.batteryConfig()
```

Comment:
`iasZoneConfig` also calls `enrollResponse`. I think at the very least you should keep pull that into the DTH so this PR doesn't change the order of configuration related messages and only removes the reporting of the CIE Address. Looking at all the DTHs changed in this PR I think the `configure` and `refresh` functions can be optimized further to reduce redundant commands and decrease the time it takes to get current status on initial join but I think those could be handled in a separate PR to decrease risk.

Reply:
I left it out mostly because it was duplicated elsewhere.  In this case `enrollResponse` is still called on configure, but you are right that it changes the order.  I could re-arrange things to try to maintain order, but I didn't think it would be a big issue for the enroll response command to come at a different point.  Do you think we should be duplicating and sending multiple versions?

Reply:
If the device isn't enrolled in the IAS Zone before the ZoneStatus is read it returns the incorrect status. I am not sure the same issue exists with reporting of ZoneStatus but I wanted to err on the side of caution. Like I said I think these functions can be reworked more massively to improve them even further which we could do now or later, but they would need to get much more thorough testing if we change the order of the functions.